### PR TITLE
update default atlantis to 0.18.2

### PR DESCRIPTION
**Pull request by chenrui333**
relates to https://github.com/runatlantis/atlantis/pull/2018

### CODE REVIEW

File: charts/atlantis/Chart.yaml [hunk 1]
```diff
@@ -1,8 +1,8 @@
-apiVersion: v1
+0.18.2apiVersion: v1
```
Comment (lkysow) [inline charts/atlantis/Chart.yaml:1]: @chenrui333 this is malformed

File: charts/atlantis/values.yaml [hunk 2]
```diff
@@ -1,4 +1,4 @@
-## -------------------------- ##
+0.18.2## -------------------------- ##
```
Comment (lkysow) [inline charts/atlantis/values.yaml:1]: same here

Comment (chenrui333) [reply charts/atlantis/values.yaml:1]: oh, i used the vscode replace. let me correct it

Comment (chenrui333) [reply charts/atlantis/values.yaml]: should be good now (codespace is still not ideal for such operations 😮‍💨 )

**Review by lkysow — APPROVED** [review]

**Pull request merged**